cloudflarekv/config.go

```diff
@@ -0,0 +1,66 @@
+package cloudflarekv
```

Comment:
@ReneWerner87 @efectn The lib is `cloudflare-go` we should just name this `cloudflare`? I'm not sold on the `cloudflarekv` name 😂

Reply:
Maybe. They name it as cloudflare workerks kv https://developers.cloudflare.com/kv/